### PR TITLE
Add Ukrainian slugs to other lang.

### PR DESCRIPTION
**Pull request by denysdovhan**
Same for https://github.com/workshopper/workshopper-adventure/pull/12 but for `workshopper`.

### CODE REVIEW

File: i18n/ko.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "pt-br": "포르투갈어(Português)",
     "vi": "베트남어(Tiếng Việt)",
     "ru": "러시아어(Русский)",
+    "uk": "우크라이나의 (Українська)",
```
Comment (marocchino) [inline i18n/ko.json]: 우크라이나어(Українська)

Comment (denysdovhan) [inline i18n/ko.json]: Fixed.

**Comment by marocchino** [comment]
Thanks!

**Comment by denysdovhan** [comment]
:+1: thnx you

**Pull request merged**